xml/docker_concept.xml

```diff
@@ -0,0 +1,498 @@
+      it somehow.</remark> used to create a
+      <emphasis>virtual machine</emphasis> on the host server. A &docker; image is
+      made by a series of layers built one over the other. Each layer
+      corresponds to a permanent change, for example an update of an
+      application.
+      <remark>toms 2019-02-04: The following sentence is dangerous and
+      misleading IMHO. The term "changes" does not relate to a Dockerfile.
+      That file is used merely as a kind of "build recipe".
+      I would propose to rephase/move the sentence at the beginning of this
+      paragraph:
+      "An image is a read-only template used to create a virtual machine on
+      the host server. Such an image is represented/build by using a
+      Dockerfile which acts as an "recipe". A Docker image is..."
+      </remark>
+      The changes are stored in a file called a
+      <emphasis>Dockerfile</emphasis>. For more details see
+      <link xlink:href="http://docs.docker.com/engine/reference/glossary#image">the
+       official Docker documentation</link>.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Dockerfile</term>
+    <listitem>
+     <para>
+      <remark>toms 2019-02-04: The description is misleading IMHO. It gives
+       it a different spin. ;)
+      According to https://docs.docker.com/engine/reference/builder/
+      "A Dockerfile is a text document that contains all the commands a user
+      could call on the command line to assemble an image. Using docker build
+      users can create an automated build that executes several command-line
+      instructions in succession."
+      I would suggest to rephrase your explanation.
+      </remark>
+      A <emphasis>Dockerfile</emphasis> stores changes made on top of the base
+      image. The <xref linkend="Docker_engine"/> reads instructions in the
+      <emphasis>Dockerfile</emphasis> and builds a new image according to the
+      instructions.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Container</term>
+    <listitem>
+     <remark>toms 2019-02-04: according to
+      https://docs.docker.com/glossary/?term=container a container consists
+      of a) a Docker image, b) an executation environment, and c) a set of
+      instructions. Maybe add that info after your first sentence?
+     </remark>
+     <para>
+      A <emphasis>container</emphasis> is a running<remark>toms 2019-02-04:
+       and a idle/stopped image is not a container? ;) Isn't that term a bit
+       too restrictive?</remark> instance based on a
+      particular &docker; <xref linkend="Docker_Image"/>. Each
+      <emphasis>container</emphasis> can be distinguished by a unique container
+      ID.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Registry</term>
+    <listitem>
+     <para>
+      A <emphasis>registry</emphasis> is storage for already created images. It
```

Comment:
Shorten to: "A storage for already created images."

Reply:
" A registry is storage and content distribution for container images" <-- it does more than just store images. It's used to store, catalog, and retrieve images.